lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -71,14 +71,14 @@ class window.Turbolinks
 
     rememberReferer()
 
-    options.partialReplace ?= false
-    options.onlyKeys ?= []
-    options.onLoadFunction = ->
-      resetScrollPosition() unless options.onlyKeys.length
-      options.callback?()
-
     fetchReplacement url, options
 
+  isPartialReplace = (options, response) ->
```

Comment:
I wonder if `isPartialReplace` might better belong inside of `Response`, or not. It seems like `turbolinks.coffee` only instantiates it when `options` is also in scope, so `options` could be passed in too.  Is it a Response responsibility?